docs/upgrading.md

```diff
@@ -4,6 +4,10 @@ description: Learn how to upgrade Next.js.
 
 # Upgrade Guide
 
+## Upgrading to 12.2
```

Comment:
This seems unrelated.

Reply:
It is but I'm sneaking it in here 😂 I noticed this was missing and likely good to have.